core/upgrade/src/test/resources/etc/examples/jetty.xml

```diff
@@ -32,7 +32,7 @@
 		<Arg>
 			<New class="org.eclipse.jetty.server.ssl.SslSelectChannelConnector">
 				<Arg>
-					<New class="org.eclipse.jetty.http.ssl.SslContextFactory">
+					<New class="org.eclipse.jetty.http.ssl.SslContextFactory$Server">
```

Comment:
@RangerRick @jeffgdotorg Just as a heads up. It seems like this is something we cannot cleanly merge forward to 26.x. The `etc/examples/jetty.xml` has changes preventing merging forward. I've just noticed this by testing with 26.1.3 and here is what I did to get it working:

```
root@ubuntu:/usr/share/opennms/etc# diff jetty.xml examples/jetty.xml
42a43
>   <!--
56c57
<   <New id="sslContextFactory" class="org.eclipse.jetty.util.ssl.SslContextFactory$Server">
---
>   <New id="sslContextFactory" class="org.eclipse.jetty.util.ssl.SslContextFactory">
114a116
>   -->
```

I've tested with Let's Encrypt certificates in the Java Keystore file and everything worked fine.

Reply:
Other than that 👍  from me.